src/doc/rustc/src/platform-support/wasm64-unknown-unknown.md

```diff
@@ -0,0 +1,101 @@
+# `wasm64-unknown-unknown`
+
+**Tier: 3**
+
+WebAssembly target which uses 64-bit memories, relying on the [memory64]
+WebAssembly proposal.
+
+[memory64]: https://github.com/webassembly/memory64
+
+## Target maintainers
+
+- Alex Crichton, https://github.com/alexcrichton
```

Comment:
I didn't realize this documentation existed before, and I think it's great to have! I also don't mean to usurp a role of sole maintainer here, if others would like to be added I'd be happy to add as part of this PR